### ISSUE TITLE
ntfs larger games

### ISSUE THREAD

**Issue by GoogleCodeExporter** [WhiteNeo/wiiflow#197]
```
What steps will reproduce the problem?
1.try to load any game over 4gig
2.
3.

What is the expected output? What do you see instead?
game should load but i get code dump 

What version of the product are you using? On what operating system?
r43 ios249

Please provide any additional information below.
i am on ntfs partition and any game over 4 gig gives me code dump and on usb 
loader gx i get this error     
's error: hd num sector doesn't match

any help?
```

Original issue reported on code.google.com by `mrnewspa...@yahoo.com` on 22 Jun 2010 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
games >4gb has been fixed as of Wiiflow 2.0, please update with a fresh wiiflow 
install .
```

Original comment by `Miigotu` on 22 Jun 2010 at 9:24
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
trying getting wiiflow 2.0 which is R45 it should fix your issues.
```

Original comment by `icpma...@gmail.com` on 22 Jun 2010 at 9:24
